Cargo.toml

```diff
@@ -13,6 +13,8 @@ edition = "2024"
 # Enable clippy's pedantic group
 [lints.clippy]
 pedantic = { level = "warn", priority = -1 }
+# Slice indexing can always panic. Why only warn in match blocks?
+match-on-vec-items = "allow"
```

Comment:
Is there a reason why you're setting these things in the `Cargo.toml` instead of at the top of main? If it's just stylistic then it's fine, but I don't think I've seen it before

Reply:
Doing it here makes it apply for things in `examples/` and `tests/` as well. We don't have anything there, but once we add integration tests, we will.

Reply:
Oh ok, I didn't know that. That makes sense.